### DISCUSSION TITLE
广告屏蔽大师 Plus+ 和 Adblock Plus+ 如何结合 OpenClash 使用？

### DISCUSSION THREAD

**Discussion by palemoky** [vernesong/OpenClash#2024] [Q&A]
我看 OpenClash 的 DNS 设置中启用本地 DNS 劫持后，是会作为 Dnsmasq 的上游服务器的，但广告屏蔽大师 Plus+ 和 Adblock Plus+ 里没有把 OpenClash 作为上游服务器的选项，请问如何设置过滤广告的功能呢？这是我的配置
![image](https://user-images.githubusercontent.com/21208635/152804232-f664d6ee-a0da-4a0c-9c36-80ac8ed69ab0.png)

我尝试取消本地 DNS 劫持，并使用自定义的上游 DNS 服务器、注释防火墙 53 端口的转发时，软路由无法打开 Google 了，重新使用本地 DNS 劫持才又能打开 Google

**Comment by Pacio** [comment]
默认的lhie1规则已经集成了dns去广告功能，和这2个插件功能重叠了，不需要开启

**Reply by palemoky** [reply]
我看到自带的 lhie1 规则了，但我看好像都没有生效，这个应该不需要再配置了吧？

**Reply by Pacio** [reply]
> 我看到自带的 lhie1 规则了，但我看好像都没有生效，这个应该不需要再配置了吧？

基于dns的去广告基本就等同于一个垃圾网站过滤器而已，https的网站都是去不了广告的，如果你想要去除那种视频网站的广告肯定是不行的，要用ikoolproxy插件

**Reply by palemoky** [reply]
那 OpenClash 可以和 ikoolproxy 一起使用吗？需要设置 ikoolproxy 上游 DNS 为 OpenClash 吗？我看有些网站的广告是有专属域名的，所以即使是 https，直接拦截掉广告的域名也是 OK 的吧？
